Universal Profiling: add integration installation (#2920)
elastic/observability-docs: docs/en/observability/profiling-get-started.asciidoc

> [[profiling-get-started]]

Author: I think they were automatically by my editor since carriage returns aren't usually in source code unless explicitly needed for Windows. I added them back to reduce the diff size for now. Aside from the profiling docs, I didn't see carriage returns in other document files.

= Get started with Universal Profiling

++++
<titleabbrev>Get started</titleabbrev>
++++

beta::[]

On this page, you'll learn how to configure and use Universal Profiling. This page covers:

* Prerequisites to setting up Universal Profiling
* Setting up Universal Profiling in your {ecloud} deployment
* Installing the host-agent

> 

Author: Installing the host agent manually (sometimes referred to as "standalone mode") is separate from installing the Universal Profiling Agent integration, which installs the host agent without manual intervention).

I added this distinction to the new section and can add similar verbiage to this section tomorrow.

You can also find these instructions by clicking the *Add data* button in the top-right corner of the page.

The following is an example of the provided instructions for {k8s}:

[role="screenshot"]
image::images/profiling-k8s-hostagent.png[]

[discrete]
[[profiling-agent-config-notes]]
=== Host-agent configuration notes

Consider the following when configuring your host-agent:

* The instructions in Kibana work well for testing environments. For production environments, we recommend setting an immutable version.

* The host-agent versioning scheme is **not aligned with the {stack} version scheme**.

* The OS packages downloaded from `releases.prodfiler.com` have a version in their file name.

* You can find a list of container image versions in the
https://container-library.elastic.co/r/observability/profiling-agent[Elastic container library repository].

* For {k8s} deployments, the Helm chart version is already used to configure the same container image, unless
overwritten with the `version` parameter in the Helm values file.

[discrete]
[[profiling-install-integration]]
== Install the Universal Profiling Agent integration

> 

Reviewer: I often see a headline without text as a lost opportunity. Can we add some text here?
E.g. we could point out the multiple ways our solution can be installed.
```
{kib} -> > **Universal Profiliing** -> **Add data**
```

[discrete]
[[find-apm-configuration]]

> === Find APM configuration

Reviewer 2: ```suggestion
=== Collect APM configuration information
```

{ecloud} runs a hosted version of {integrations-server} that includes the APM integration.

. In {kib}, navigate to **Fleet** > **Agent policies** and select the **Elastic Cloud agent policy**.

. Click **Elastic APM** in the **Names** column.

. Inside the **General** > **Server configuration** section, write down the value in the **URL** field.

. Inside the **Agent authorization** > **Maximum number of API keys of Agent authentication** section, write down the value in the **Secret token** field.

[discrete]
[[add-integration]]
=== Add integration

> . Navigate to **Management** > **Integrations**.

Reviewer 2: ```suggestion
. Under **Management** in the left navigation, select **Integrations**.
```

> . Toggle **Display beta integrations** to the on position in the left sidebar.

Reviewer 2: ```suggestion
. Turn on **Display beta integrations** in the left sidebar.
```

> . In the **Search for integrations** text field, enter `Universal`.

Reviewer: Searching for `Universal` returns two results. `Universal Profiling Agent` and `Universal Profiling Symbolizer` (and soonish `Universal Profiling Collector`). From an outside perspective these integrations look identical besides the name. So I think it is important to point out the differences and maybe explain the reasoning?

Author: For the moment, I added an exact search term so that only one result appears. I agree we should add an explanation for all of our integrations, but perhaps it should be in another section where it could get a more thorough explanation.

> . Click on the **Universal Profiling Agent** card.

Reviewer 2: ```suggestion
. Select the **Universal Profiling Agent** card.
```

. Click **Add Universal Profiling Agent**.

> . Inside the **Universal Profiling Agent** > **Settings** section, add the URL from <<find-apm-configuration>> above to the **Universal Profiling collector endpoint** field.

Reviewer 2: ```suggestion
. In **Universal Profiling Agent → Settings**, add the information you collected from the  <<find-apm-configuration>> section: 
.. Add the URL from <<find-apm-configuration>> to the **Universal Profiling collector endpoint** field.
.. Add the secret token from <<find-apm-configuration>> to the **Authorization** field.
```

> . Inside the **Universal Profiling Agent** > **Settings** section, add the secret token from <<find-apm-configuration>> above to the **Authorization** field.
> 

Reviewer 2: ```suggestion
```
